windows/src/desktop/kmshell/locale/mnw-MM/strings.xml

```diff
@@ -465,7 +465,7 @@
   <!-- Context: Configuration Dialog - Update tab -->
   <!-- String Type: PlainText -->
   <!-- Introduced: 18.0.168 -->
-  <string name="S_Updates_Available" comment="Update tab - informs user updates are available ">သမၠုၚ်ပ္တိုန်တံဂှ် ယဝ်ကၟာတ်ပံက်ထောံ မုက်လိက်တုဲမ္ဂး ဒှ်ဒတန်ရ။:</string>
+  <string name="S_Updates_Available" comment="Update tab - informs user updates are available ">သမၠုၚ်ပ္တိုန်တံဂှ် ယဝ်ကၟာတ်ပံက်ထောံ မုက်လိက်တုဲမ္ဂး ဒှ်ဒတန်ရ။</string>
   <!-- Context: Configuration Dialog - Update tab pop up -->
```

Comment:
The colon is there again in the Beta branch. A pain I know.

Reply:
The colon shouldn't be there, right?

Reply:
The colon is in the English it was introduced in PR #13555. So it should be there.

Reply:
ah, this is a case of simultaneous changes. #13555 tweaked the string to expect a colon, which Crowdin now is synced to expect. I believe as of now, this Mon string is for the previous translation (and is now unapproved in crowdin).  It will get updated next time the crowdin translators get to it.